### PR TITLE
Setup Flask Project Structure

### PR DESCRIPTION
**Pull request by McIntireEvan**
* Create virtualenv
* Create flask app, files for routes
* Create Instructions of running

### CODE REVIEW

**Comment by McIntireEvan** [comment]
I have a few more details I want to add to the DEVELOPING.md (specifically about using blueprints), and some rewording, but I just wanted to get this out here so we can adjust as needed right now

File: api/db.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# Database access
```
Comment (timothychen01) [inline api/db.py]: Would this be where we connect to the DB? (And in turn reference a secret .config file that we .gitignore?)

Comment (McIntireEvan) [reply api/db.py]: Yes :) Thought I made it, but guess I didn't

**Comment by McIntireEvan** [comment]
Added a bit more, let me know if that's good enough to start - I think it should be pretty easy for everyone to work with, and if it turns out to be harder, we can look at the structure again

I want to get unit tests added, but I'm a bit pressed for time this week and python modules aren't my specialty - but I don't want to be blocking anyone

**Comment by timothychen01** [comment]
LGTM!

**Pull request merged**